src/main/drivers/dma_reqmap.c

```diff
@@ -92,10 +92,6 @@ static const dmaPeripheralMapping_t dmaPeripheralMapping[] = {
     REQMAP(ADC, 3),
 #endif
 
-#ifdef USE_SDCARD_SDIO
-    REQMAP_SGL(SDIO),
```

Comment:
Why was this removed?

Reply:
because H7 uses IDMA for SDCARD, see functional block diagram in reference manual.